docs/development.md

```diff
@@ -188,6 +189,13 @@ KubeFed controller manager:
 
 Follow the [cleanup instructions in the user guide](../charts/kubefed/README.md#uninstalling-the-chart).
 
+## Embedding static files using go-bindata
+This project is using `go-bindata` tool for embedding static files into its e2e test-suite to enable the creation of a self-contained e2e binary.
+
+You can install this utility using the [download-binaries.sh](../scripts/download-binaries.sh) script.
+
+Use `make generate` to regenerate the `bindata.go` file in case the bundled content changes. It's necessary to follow this step to ensure that e2e test-suite passes the CI build. Please refer to [this](../scripts/update-bindata.sh) script for more information.
```

Comment:
Please adjust your editor to format the lengths of these lines to match the rest of the file.

Reply:
Fixed.